### DISCUSSION TITLE
SAM problem

### DISCUSSION THREAD

**Discussion by Medeir0s95** [kubilus1/autoortho#447] [General]
Hello there. When i'm install SAM in my XP12 and 11, my ortho had a problem when i opened the simulator like that. Someone knows how can i solve that??? 

![Uploading image.png…]()

**Comment by Medeir0s95** [comment]
I'm used the version 0.7.1 the autortho.

**Comment by Medeir0s95** [comment]
![image](https://github.com/kubilus1/autoortho/assets/137718884/a696c477-0ba7-437f-8752-a894f49f17f6)

**Comment by chezhj** [comment]
What's the problem & include ini, logs etc
